controllers/dpa_controller.go

```diff
@@ -102,6 +102,7 @@ func (r *DPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 		r.ReconcileResticDaemonset,
 		r.ReconcileVeleroServiceMonitor,
 		r.ReconcileVeleroMetricsSVC,
+		r.ReconcileDataMoverController,
```

Comment:
Are we adding datamover by default?

Reply:
Just for this PR, did not want to post a long PR, I have a [TODO ](https://github.com/openshift/oadp-operator/pull/727/files#diff-a97f576da4c6bee59ebf0b5c4462c39c6eb7d1906bd517dbcef41acdec0263a0R30)added and will be addressed in the plugin install PR which will be based on `datamover` as a default plugin listed on the DPA CR

Reply:
shots fired.